libs/jsonschema/schema.go

```diff
@@ -20,6 +20,10 @@ type Schema struct {
 	// IDE. This is manually injected here using schema.Docs
 	Description string `json:"description,omitempty"`
 
+	// Expected value for the object. Validation succeeds if the instance is equal
+	// to this keyword's value.
+	Const any `json:"const,omitempty"`
```

Comment:
Can you elaborate on the comment? I.e. the value must be equal to this field?

Reply:
done